edb/lib/ext/auth.edgeql

```diff
@@ -30,6 +30,19 @@ CREATE EXTENSION PACKAGE auth VERSION '1.0' {
         create constraint exclusive on ((.iss, .sub))
     };
 
+    create type ext::auth::LocalIdentity {
+        create required property handle: std::str {
```

Comment:
How do we feel about the term `handle` here? This property is what will be provided as the lookup for a set of credentials. So for password flow, it'll be an email or username. For something like WebAuthn, it'll be a [credential ID](https://developer.mozilla.org/en-US/docs/Web/API/PublicKeyCredential/id).

Here are some alternatives:

1. login
2. key
3. identifier
4. alias
5. ??

Reply:
IMO, `handle` is most neutral and unambiguous.